test/restify/app/src/app.ts

```diff
@@ -6,6 +6,7 @@ function respond(req: any, res: any, next: any) {
 }
 
 const server = restify.createServer()
+server.use(restify.plugins.queryParser())
```

Comment:
Would it make sense for the integration test to look for the presence of the `appsignal.request.params` attribute?

Reply:
I've updated the tests for it.